dags/mapping-elt.py

```diff
@@ -1,55 +1,68 @@
-from airflow import DAG
```

Comment:
Can't comment on the file directly, but we can remove the `example-dag-basic` since that's no longer part of the CLI project :)

Reply:
Already removed it :)